### PR TITLE
Fix GraphQL spec

### PR DESCRIPTION
**Pull request by mikker**

### CODE REVIEW

File: spec/integration/graphql_spec.rb [hunk 1]
```diff
@@ -35,7 +41,7 @@
 
     def setup_database
       ActiveRecord::Base.logger = Logger.new(SpecLogger)
-      ActiveRecord::Base.logger = Logger.new(nil)
+      # ActiveRecord::Base.logger = Logger.new(nil)
```
Comment (mikker) [inline spec/integration/graphql_spec.rb:44]:
```suggestion
      ActiveRecord::Base.logger = Logger.new(nil)
```

File: spec/integration/graphql_spec.rb [hunk 2]
```diff
@@ -132,35 +134,33 @@ def index
...
-            json: { error: { message: e.message }, data: {} }
-          )
+        # rescue StandardError => e
+        #   logger.error e.message
+        #   logger.debug e.backtrace
+
+        #   render(
+        #     status: 500,
+        #     json: { error: { message: e.message }, data: {} }
+        #   )
```
Comment (mikker) [inline spec/integration/graphql_spec.rb:163]:
```suggestion
        rescue StandardError => e
          logger.error e.message

          render(
            status: 500,
            json: { error: { message: e.message }, data: {} }
          )
```

File: spec/integration/graphql_spec.rb [hunk 3]
```diff
@@ -132,24 +134,21 @@ def index
...
-          context = {}
+          context_ = {}
 
           result =
-            if (multi = params[:queries])
+            if (multi = params[:multi])
               Types::GraphQLTestAppSchema.multiplex(
                 multi.map do |q|
-                  { query: q, variables: variables, context: context }
+                  { query: q[:query], variables: q[:variables], context: context_.dup }
```
Comment (mikker) [inline spec/integration/graphql_spec.rb:143]:
```suggestion
                  { query: q[:query], variables: q[:variables], context: context_ }
```

**Comment by estolfo** [comment]
Copying what I wrote in slack: I don’t know where the source is exactly yet but it’s from this statement in the graphql spec: `Post.find_by(slug: slug)`. So I’m guessing it’s caused by a recent change between v6.1.1 and 6.1.2 to `ActiveRecord`. You could change it to `Post.where(slug: slug).first` and the test passes.

File: spec/integration/graphql_spec.rb [hunk 4]
```diff
@@ -89,16 +89,12 @@ def posts
           end
 
           def post(slug:)
-            Post.find_by(slug: slug)
+            Post.find_by(slug: slug).first
```
Comment (estolfo) [inline spec/integration/graphql_spec.rb:92]: If you do `Post.where(slug: slug).first`, it will work

**Review by estolfo — CHANGES_REQUESTED** [review]

**Review by estolfo — APPROVED** [review]

**Pull request merged**